tests/modules/bamtools/convert/main.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
```

Comment:
You also only have one test - I would test for all supported file conversions

Reply:
Done, I added a test for each supported format.

Reply:
Nice